src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -2,20 +2,20 @@
   <ItemGroup>
     <RuntimeGroup Include="unix">
       <Parent>any</Parent>
-      <Architectures>x64;x86;arm;armel;arm64;mips64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;mips64;s390x</Architectures>
     </RuntimeGroup>
     <RuntimeGroup Include="linux">
       <Parent>unix</Parent>
-      <Architectures>x64;x86;arm;armel;arm64;mips64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;mips64;s390x</Architectures>
     </RuntimeGroup>
     <RuntimeGroup Include="linux-musl">
       <Parent>linux</Parent>
-      <Architectures>x64;x86;arm;armel;arm64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;s390x</Architectures>
```

Comment:
Similar question - does musl targeting s390x really exist?

Reply:
It's really only used for alpine, but yes, musl also supports s390x:
https://www.musl-libc.org/intro.html